encoding/gocode/binding.tmpl

```diff
@@ -5,7 +5,8 @@ import (
 	"io/fs"
 	"path/filepath"
 
-    "cuelang.org/go/cue/build"
+	{{ if .CUEPath }}"cuelang.org/go/cue"{{ end }}
```

Comment:
...this is what i should have suggested 😄 
```suggestion
	"cuelang.org/go/cue"
```

Reply:
That's not better - if we aren't setting the cue path, we're not adding a line with cue.Path, so that import will be removed.

Reply:
right, this is more of a consistency nit than anything. There are other imports in that list - at least `embed` and `io/fs` - which are only conditionally necessary, as well. But they're not wrapped in a conditional.

The conclusion i came to after writing a few of these templates is that it's easier to maintain a template with the full set of imports that may be necessary and rely on `goimports` to remove unnecessary ones for particular cases, than to have the additional conditionals in the template file.

Reply:
Ok, I can do that. Can I remove the warning about relying on goimports, if that's now how expected?

Reply:
And yet, when I ran the command, I got a warning, which is why I made the change. I'd like to either add the import or remove the warning.